src/constant.ts

```diff
@@ -1,8 +1,91 @@
+import { BankDataRecord } from './types';
+import BC_CARD from './assets/bank/bc_card.png';
+import SHINHAN_CARD from './assets/bank/shinhan_card.png';
+import KAKAO_BANK from './assets/bank/kakao_bank.png';
+import HYUNDAI_CARD from './assets/bank/hyundai_card.png';
+import WOORI_CARD from './assets/bank/woori_card.png';
+import LOTTE_CARD from './assets/bank/lotte_card.png';
+import HANA_CARD from './assets/bank/hana_card.png';
+import KB_CARD from './assets/bank/kb_card.png';
+
 export const LOCAL_STORAGE_CARD_KEY = 'cardList';
 
-export const HOME_PAGE = 'homePage';
-export const ADD_CARD_PAGE = 'addCardPage';
+export const PAGE_KIND = Object.freeze({
+  home: 'homePage',
+  addCard: 'addCardPage',
+});
+
+export const PAGE = [PAGE_KIND.home, PAGE_KIND.addCard] as const;
 
-export const PAGE = [HOME_PAGE, ADD_CARD_PAGE] as const;
+export const TAB_INDEX_INFO = {
+  dismiss: 999,
```

Comment:
요건 그냥 의견인데 상수로 쓰이는 값들이 아래처럼 대문자로 관리되면 코드를 볼 때 상수임을 한 눈에 확인할 수 있어서 좋을거 같아요

```suggestion
export const TAB_INDEX_INFO = {
  DISMISS: 999,
```